### ISSUE TITLE
two hop (both ends pinned) not finding results.

### ISSUE THREAD

**Issue by cbizon** [ranking-agent/strider#210]
```
{
    "message": {
        "query_graph": {
            "nodes": {
                "n0": {
                    "ids": [
                        "MONDO:0007743"
                    ],
                    "categories": [
                        "biolink:Disease"
                    ]
                },
                "n1": {
                    "categories": [
                        "biolink:Gene"
                    ]
                },
                "n2": {
                    "ids": [
                        "CHEBI:31859"
                    ],
                    "categories": [
                        "biolink:ChemicalSubstance"
                    ]
                }
            },
            "edges": {
                "e0": {
                    "subject": "n0",
                    "object": "n1"
                },
                "e1": {
                    "subject": "n2",
                    "object": "n1"
                }
            }
        }
    }
}
```

https://github.com/NCATSTranslator/testing/issues/77

I think that this is showing up as a timeout in the ARS, as strider takes 7+ minutes to complete.

When strider is complete, it is not returning any results.  There is an expected result of gene: SLC 6A3

**Comment by cbizon** [comment]
We are getting 504 on robokopkg and 400 for CAM KP

504 Server Error: Gateway Time-out for url: https://automat.renci.org/robokopkg/1.1/query
For more information check: https://httpstatuses.com/504 (830 times)
400 Client Error: Bad Request for url: https://cam-kp-api-dev.renci.org/query
For more information check: https://httpstatuses.com/400 (162 times)

**Comment by cbizon** [comment]
Closed as dup of #222
